indico/modules/designer/util.py

```diff
@@ -38,3 +38,17 @@ def get_all_templates(obj):
 def get_inherited_templates(obj):
     """Get all templates inherited by a given event/category"""
     return get_all_templates(obj) - set(obj.designer_templates)
+
+
+def get_default_template_on_category(obj):
+    if isinstance(obj, Event):
```

Comment:
I believe you've added this here as a way to protect it from the case where `_render_template_list` calls it with an event. But I don't think it makes much sense to plan for a case like this in a function that is `_on_category`. I would instead do the check on the calling function(s).